eigen_cpp/lib/dune

```diff
@@ -0,0 +1,16 @@
+(ignored_subdirs
+ (Eigen unsupported)
+)
+
+(rule
+ (targets libeigen.a libeigen.so)
+ (deps
+  (source_tree lib)
```

Comment:
I think this line is incorrect (and the compilation fails for me here). Shouldn't it be: `(source_tree .)`?

Reply:
Yes, it should :)

Reply:
Ops, I forgot to commit the fix, sorry